samcli/lib/build/app_builder.py

```diff
@@ -679,8 +685,11 @@ def _get_build_options(
             build_props = metadata.get(BUILD_PROPERTIES, {})
             # Esbuild takes an array of entry points from which to start bundling
             # as a required argument. This corresponds to the lambda function handler.
-            if handler and not build_props.get("EntryPoints"):
-                entry_points = [handler.split(".")[0]]
+            if functions and not build_props.get("EntryPoints"):
```

Comment:
are the build properties metadata properties added to each lambda function that will use ESBuild?